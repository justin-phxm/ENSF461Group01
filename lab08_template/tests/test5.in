108
714
45
632
67
566
141
534
303
316
323
44
179
645
97
348
