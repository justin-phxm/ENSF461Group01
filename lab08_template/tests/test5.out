108
822
867
1499
1566
2132
2273
2807
3110
3426
3749
3793
3972
4617
4714
5062
